packages/typespec-python/test/azure/generated/resiliency-srv-driven2/resiliency/srv/driven2/_operations/_operations.py

```diff
@@ -209,9 +206,6 @@ def from_none(self, *, new_parameter: Optional[str] = None, **kwargs: Any) -> bo
         return 200 <= response.status_code <= 299
 
     @distributed_trace
-    @api_version_validation(
-        params_added_on={"v2": ["new_parameter"]},
-    )  # pylint: disable=inconsistent-return-statements
     def from_one_required(  # pylint: disable=inconsistent-return-statements
```

Comment:
can be partly fixed by https://github.com/Azure/autorest.python/pull/2580.
related TCGC issue https://github.com/Azure/typespec-azure/issues/802